### DISCUSSION TITLE
Add proxy support

### DISCUSSION THREAD

**Discussion by kofany** [squidowl/halloy#223] [General]
Can You add proxy support, to connect with irc networks over proxy?

**Comment by kofany** [comment]
+ /whois should have also a second parameter /whois nick nick shows also idle or implementation of /wii command  like is done in irssi irc client :)

**Reply by casperstorm** [reply]
Could you create a issue with this?
It sounds like a new feature request :)

**Comment by kofany** [comment]
Ok as for now this client is no usable for example after banhick he still is on channel, no info about ban/kick no info about channel exit.

**Reply by casperstorm** [reply]
Could you create a issue with this, explaining what your expected and what you see? Easier to track and work on it then :)

**Comment by casperstorm** [comment]
Proxy support has been added.
